packages/tocco-ui/src/Upload/UploadInput.js

```diff
@@ -25,6 +25,20 @@ const UploadInput = ({onDrop: onDropProp, immutable, text}) => {
               position={design.position.PREPEND}
             />
           </Button>
+          {onChoose && <Button
+
+            onClick={
+              e => {
+                onChoose()
+                e.stopPropagation()
+              }
+            }
+          >
+            <Icon
+              icon="folder"
+              position={design.position.PREPEND}
+            />
```

Comment:
Now that there are two icon buttons, i think they both should have a title that is shown on mouseover (e.g. "choose from computer" and "choose from uploaded documents" or something like that)

Reply:
just setting a title on the icon does not work. there is a new task for yilmaz to fix several styling things

Reply:
what about the `title` attribute of the `Button`? doesn't work either?

Reply:
also doesn't work